packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js

```diff
@@ -1862,6 +1862,41 @@ describe('plugin-meetings', () => {
           assert.strictEqual(mqeData.audioReceive[0].streams.length, 0);
         });
       });
+
+      describe('window and screen size emission', async () => {
+        beforeEach(async() => {
+          await startStatsAnalyzer();
+        })
+
+        it('should record the screen size from window.screen properties', async () => {
+          sinon.stub(window.screen, 'width').get(() => 1280);
+          sinon.stub(window.screen, 'height').get(() => 720);
+          await progressTime(MQA_INTERVAL);
+          assert.strictEqual(mqeData.intervalMetadata.screenWidth, 1280);
+          assert.strictEqual(mqeData.intervalMetadata.screenHeight, 720);
+          assert.strictEqual(mqeData.intervalMetadata.screenResolution, 3600);
+        })
+
+        context('when app window size changes', () => {
+          it('should record the initial app window size from window properties', async () => {
```

Comment:
These 2 tests seem to be the same.. in order to test if window size changes are being recorded, would it be better if we do this within one test itself where the `get()` function returns different values for the first and second times it's called?